USAGE.md

```diff
@@ -19,77 +19,358 @@ Now `make install` only supports Linux Ubuntu, CentOS and Mac OS X. If you use o
 you should follow [INSTALL.md](https://github.com/facebook/rocksdb/blob/master/INSTALL.md) to
```

Comment:
1. In the previous line, please change "Notice" to "Note".
2. Change this sentence to "install RocksDB manually first according to [INSTALL.md](https://github.com/facebook/rocksdb/blob/master/INSTALL.md)."